jmx_prometheus_httpserver/src/main/java/io/prometheus/jmx/WebServer.java

```diff
@@ -18,24 +14,14 @@ public static void main(String[] args) throws Exception {
 
      String[] hostnamePort = args[0].split(":");
      int port;
-     InetSocketAddress socket;
 
      if (hostnamePort.length == 2) {
        port = Integer.parseInt(hostnamePort[1]);
-       socket = new InetSocketAddress(hostnamePort[0], port);
```

Comment:
Sure, I will put it back once HTTP server is able to take a host and port. I can do the modification and test, if you want.